lua/lvim/plugin-loader.lua

```diff
@@ -3,174 +3,178 @@ local plugin_loader = {}
 local utils = require "lvim.utils"
 local Log = require "lvim.core.log"
 local join_paths = utils.join_paths
-local in_headless = #vim.api.nvim_list_uis() == 0
 
--- we need to reuse this outside of init()
-local compile_path = join_paths(get_config_dir(), "plugin", "packer_compiled.lua")
-local snapshot_path = join_paths(get_cache_dir(), "snapshots")
-local default_snapshot = join_paths(get_lvim_base_dir(), "snapshots", "default.json")
+local plugins_dir = join_paths(get_runtime_dir(), "lazy", "plugins")
+
+local function remove_rtp_paths()
```

Comment:
not needed with lazy. lazy already resets the rtp

Reply:
it adds `vim.fn.stdpath("config")`, which is undesirable in lunarvim, but I've realized that my removing from rtp doesn't take effect, since lazy resets it after I remove the paths and loads .config/nvim/after anyway
@kylo252 what about adding `config` to [our override of vim.fn.stdpath](https://github.com/LunarVim/LunarVim/blob/62c2c0e15706be48abd2e3d353aaf79b00548d58/lua/lvim/bootstrap.lua#L66-L71)?

Reply:
you can disable rtp reset or add extra paths if you like. check the docs on config

Reply:
I'll probably do that, I though it would be nice to have it all in lazy.nvim, but lunarvim's use case is quite unusual. Thank you for taking the time to review this PR